server/modules/README.md

```diff
@@ -0,0 +1,4 @@
+### Module Checkout Directory
+
+Clone module source into this directory. Gradle will look for modules here when building. \
```

Comment:
Is the backslash at the end there intended?

Reply:
Yeah. It adds a line break in markdown.